core/contracts/test/FixedPointTest.sol

```diff
@@ -17,10 +17,18 @@ contract FixedPointTest {
         return FixedPoint.Unsigned(a).isGreaterThan(FixedPoint.Unsigned(b));
     }
 
+    function wrapMixedIsGreaterThan(uint a, uint b) external pure returns (bool) {
```

Comment:
Are you planning to test the reverse? Something like:
```
return a.isGreaterThan(FixedPoint.Unsigned(b));
```

Reply:
oh right! added